compiler/test/dotty/partest/DPConsoleRunner.scala

```diff
@@ -105,9 +105,9 @@ extends SuiteRunner(testSourcePath, fileManager, updateCheck, failed, javaCmdPat
     val seqResults =
       if (!sequentialTests.isEmpty) {
         val savedThreads = sys.props("partest.threads")
-        sys.props("partest.threads") = "1"
+        sys.props("partest.threads") = "2"
 
-        NestUI.echo(s"## we will run ${sequentialTests.length} tests sequentially")
+        NestUI.echo(s"## we will run ${sequentialTests.length} tests using ${PartestDefaults.numThreads} thread(s)")
```

Comment:
You did not update the documentation and the variable names to reflect the fact that these tests are not run sequentially anymore.